src/components/Person/Person.jsx

```diff
@@ -1 +1,45 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import Proptypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    sex,
+    age,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+
+      <p className="Person__partner">
+        {isMarried
+          ? (`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`)
```

Comment:
create a variable for the partner and use it here. It'll be more readable
```
const partner = sex === 'm' ? 'wife' : 'husband';
```

Reply:
Done!
Thanks!